plone/app/content/browser/vocabulary.py

```diff
@@ -74,6 +74,50 @@ class VocabLookupException(Exception):
 
 class BaseVocabularyView(BrowserView):
 
+    def get_translated_ignored(self):
+        return [
+            'author_name',
```

Comment:
while at it, it is probably a good idea to make the default list a module global variable like so:

    TRANSLATED_IGNORE = [
        'author_name',
        'cmf_uid',
        'commentators',
        'created',
        ...
    ]
    ...

        def get_translated_ignored(self):
            return TRANSLATED_IGNORE